modles/getModel.js

```diff
@@ -36,10 +33,8 @@ function getarticle_idMdl(id) {
       for (let i = 0; i < result.rows.length; i++) {
         Object.assign(Obj, result.rows[i]);
       }
-      return Obj;
-    })
-    .catch(() => {
-      return Promise.reject({ status: 404, msg: "not found" });
+      if (Obj === {}) return Promise.reject({ status: 404, msg: "not found" });
```

Comment:
this for loop is a little redundant as we could check the result.rows length to see if it is equal to 0